pymc/step_methods/metropolis.py

```diff
@@ -262,39 +299,10 @@ def competence(var, has_grad):
 
 def tune(scale, acc_rate):
     """
-    Tunes the scaling parameter for the proposal distribution
-    according to the acceptance rate over the last tune_interval:
-
-    Rate    Variance adaptation
-    ----    -------------------
-    <0.001        x 0.1
-    <0.05         x 0.5
-    <0.2          x 0.9
-    >0.5          x 1.1
-    >0.75         x 2
-    >0.95         x 10
-
+    Tunes the scaling parameter for the proposal distribution according to the distance
+    to 0.234 acceptance rate.
     """
-    if acc_rate < 0.001:
-        # reduce by 90 percent
-        return scale * 0.1
-    elif acc_rate < 0.05:
-        # reduce by 50 percent
-        return scale * 0.5
-    elif acc_rate < 0.2:
-        # reduce by ten percent
-        return scale * 0.9
-    elif acc_rate > 0.95:
-        # increase by factor of ten
-        return scale * 10.0
-    elif acc_rate > 0.75:
-        # increase by double
-        return scale * 2.0
-    elif acc_rate > 0.5:
-        # increase by ten percent
-        return scale * 1.1
-
-    return scale
+    return np.exp(np.log(scale) + (acc_rate - 0.234))
```

Comment:
@aloctavodia Is this correct?

Reply:
Yes, also the initial guess for the scale should be something like this https://github.com/pymc-devs/pymc/blob/5703a9d24ca01da8b81363874e023e59cd18688d/pymc/smc/smc.py#L461-L462 (for problem with a few dimensions is the same as starting from 1)

We can make the scaling changes in a different PR.

Reply:
Yeah, let's do it in a separate PR, also because of the MLDA complaining here